cmd/list_terms.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the GPLv3, see LICENCE file for details.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+	"github.com/juju/persistent-cookiejar"
+	"gopkg.in/macaroon-bakery.v1/httpbakery"
+
+	"github.com/juju/terms-client/api"
+)
+
+const (
+	listTermsDoc = `
+list-terms show a list of Terms and Conditions documents owned by you
+or one of the public groups you belong to.
+Examples
+list-terms --groups test-group1,test-group2
```

Comment:
I think we discussed this... we'll just replace it, as this command implements the currently documented behavior.